src/component/feature/FeatureToggleList/FeatureToggleListTable/FeatureSeenCell/FeatureSeenCell.tsx

```diff
@@ -0,0 +1,110 @@
+import { useStyles } from './FeatureSeenCell.styles';
+import TimeAgo from 'react-timeago';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { Tooltip } from '@mui/material';
+import React from 'react';
+
+function generateUnit(unit?: string): string {
+    switch (unit) {
+        case 'second':
+            return 's';
+        case 'minute':
+            return 'm';
+        case 'hour':
+            return 'h';
+        case 'day':
+            return 'D';
+        case 'week':
+            return 'W';
+        case 'month':
+            return 'M';
+        case 'year':
+            return 'Y';
+        default:
+            return '';
+    }
+}
+
+function getColor(unit?: string): string {
+    // FIXME: theme
+    switch (unit) {
+        case 'second':
+            return '#98E3AF';
+        case 'minute':
+            return '#98E3AF';
+        case 'hour':
+            return '#98E3AF';
+        case 'day':
+            return '#98E3AF';
```

Comment:
Should combine these cases to make it clear that their return values are the same.